packages/web/src/components/buy-sell-modal/SwapTab.tsx

```diff
@@ -27,45 +34,25 @@ const SwapFormSkeleton = () => (
   </Flex>
 )
 
+// Legacy props interface for backward compatibility
```

Comment:
just wondering about backwards compat... can we just migrate everything to new stuff?